src/Modules/OrchardCore.Commerce/Controllers/PaymentController.cs

```diff
@@ -58,7 +61,8 @@ public class PaymentController : Controller
         ISiteService siteService,
         IUpdateModelAccessor updateModelAccessor,
         IRegionService regionService,
-        Lazy<IUserService> userServiceLazy)
+        Lazy<IUserService> userServiceLazy,
+        IWorkflowManager workflowManager)
```

Comment:
If `IWorkflowManager` is not registered this will throw an exception. Services in the constructor are always mandatory. You can use `IEnumerable<IWorkflowManager>` here and `.FirstOrDefault()` below